README.md

```diff
@@ -78,7 +85,7 @@ __Note__: if you didn't specify a filename when converting, you can obtain the f
 In order to test how a URL will be converted and see the outcome, you can utilise the `/admin/tool/pdfpages/test.php` page in your browser.
 This will utilise the configured converter on the server side to carry out the conversion, creating the converted file in the Moodle file system and then serve up the PDF to the browser.
 
-In order to access this page, the logged in Moodle user needs to be an Admin or have a role with the capability `tool/pdfpages:generatepdf` at the system level.
+In order to access this page, the logged in Moodle user needs to be an Admin or have a role with the capability `tool/pdfpages:generatepdf` at the system or module context level.
```

Comment:
As with the other part of the readme, I think this may need to say something like "have a role with the capability `blah` in the relevant context".

Reply:
Updated https://github.com/catalyst/moodle-tool_pdfpages/pull/40/commits/7325fffcf13d151213ca65d758ad063c7125fbb7 xD